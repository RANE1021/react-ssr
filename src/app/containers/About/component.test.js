
import React from "react";
import { shallow } from "enzyme";

import AboutContainer from "./component";

const styles = {};

describe("The home component mounts with one div and About component", () => {
  const shallowAboutContainer = shallow(<AboutContainer styles={styles} />);
  it("loads with divs", () => {
    expect(shallowAboutContainer.find("div").length).toBe(1);
  });

  it('matches previous snapshot', () => {
    expect(AboutContainer).toMatchSnapshot()
  })
});
