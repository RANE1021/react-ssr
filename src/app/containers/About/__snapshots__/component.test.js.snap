// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The home component mounts with one div and  matches previous snapshot 1`] = `[Function]`;

exports[`The home component mounts with one div and Ab matches previous snapshot 1`] = `[Function]`;

exports[`The home component mounts with one div and About component matches previous snapshot 1`] = `[Function]`;

exports[`The home component mounts with styles and three divs matches previous snapshot 1`] = `[Function]`;
